lib/util.js

```diff
@@ -597,9 +597,8 @@ function formatValue(ctx, value, recurseTimes, ln) {
 }
 
 function formatNumber(fn, value) {
-  // Format -0 as '-0'. A `value === -0` check won't distinguish 0 from -0.
-  // Using a division check is currently faster than `Object.is(value, -0)`
-  // as of V8 6.1.
+  // `Object.is` is faster as of 6.3, and this code doesn't
+  // need to be very performant anyway
```

Comment:
Sorry, I was thinking something more along the lines of:

```
// Format -0 as '-0'. A `value === -0` check won't distinguish 0 from -0.
```

We don't really need to talk about the performance of it, IMHO.

Reply:
alright :smile: